bench/config/production_setup.py

```diff
@@ -17,9 +17,11 @@
 
 
 def setup_production_prerequisites():
-	"""Installs ansible, fail2banc, NGINX and supervisor"""
+	"""Installs ansible, gunicorn, fail2banc, NGINX and supervisor"""
 	if not which("ansible"):
 		exec_cmd(f"sudo {sys.executable} -m pip install ansible")
+	if not which("gunicorn"):
```

Comment:
Don't think you're going to find `gunicorn` in PATH. You'll have to check if gunicorn exists in env manually. Unless, I'm missing something?